fvm/evm/testutils/contract.go

```diff
@@ -379,6 +379,38 @@ func GetDummyKittyTestContract(t testing.TB) *TestContract {
 	}
 }
 
+func GetSelfDestructTestContract(tb testing.TB) *TestContract {
+	bytecode, err := hex.DecodeString("6080604052608180600f5f395ff3fe6080604052348015600e575f80fd5b50600436106026575f3560e01c806383197ef014602a575b5f80fd5b60306032565b005b3373ffffffffffffffffffffffffffffffffffffffff16fffea2646970667358221220a29c41b43845784caa870cd38e1eb6abe7b1f79a50d59ea6ae51d8337eb3b2fd64736f6c63430008160033")
+	require.NoError(tb, err)
+
+	return &TestContract{
+		Code: `
+			contract TestDestruct {
+				constructor() payable {}
+			
+				function destroy() public {
+					selfdestruct(payable(msg.sender));
+				}
```

Comment:
I suggest we add this method to the other test contract and compile it all together.

Reply:
Done [09140a7](https://github.com/onflow/flow-go/pull/5273/commits/09140a789554b34c280cd21fd7ac898835de6304)